### ISSUE TITLE
Not all non-Amazon Alexa-enabled Devices are being recognized

### ISSUE THREAD

**Issue by sgnihttrams** [tonesto7/echo-speaks#45]
NOTE: I also have 'real' Amazon Alexa devices (Echo Speaks working), AND other non-Amazon Alexa devices (Echo Speaks working), but this issue is not about any of those. 

FIrst-off, this is NOT a high priority for me. I'm just posting here to get it in the mix...

I have two computers using the Alexa app for Windows. Both are running Windows 10 (one is Home, the other is Pro). 

One of my non-Amazon Alexa-enabled devices (Alexa running in Windows 10 Home on my laptop) got a device created for it by Echo Speaks in SmartThings, but another one (Alexa running in Windows 10 Pro on my desktop) on did not.

Is that just part of the deal, and we can’t necessarily count on those kinds of devices getting reproduced via this method, or might I have something that I need to adjust or fix?

**Comment by tonesto7** [comment]
@sgnihttrams 
Can you please send me a screen shot of the Devices page in the SmartApp that lists all of there DeviceTypes and such?

I'm curious to see what your windows machine was detected as

**Comment by sgnihttrams** [comment]
Doh! 
It worked. 
I took another look in the Amazon Alexa app on my Android phone, and noticed that the Windows computer in question was listed as offline. 

I then closed the Alexa app on that computer, and reopened it, and once 'offline' went away in the Alexa Android app, I went back into the SmartApp, and there it was. I saved, and then it showed up in my list of 'things' in SmartThings.

I assume this point was already obvious and/or already stated somewhere, but I now understand that a device must be 'online' in the Alexa world at the time of running the SmartApp in order for it to get a 'thing' created for it in SmartThings.

Thanks!  :-)
